### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by swethalohith2**
## What
some enhancements 

## References
[Kanban card](https://instana.kanbanize.com/ctrl_board/132/cards/145985/details/)

### CODE REVIEW

File: synctl [hunk 1]
```diff
@@ -3467,7 +3494,7 @@ class ParseParameter:
             '--alert-channel', type=str, nargs='+', metavar="<id>", help="alert channel id, support multiple alert channel id")
 
         self.parser_create.add_argument(
-            '--severity', type=int, metavar="<int>", choices=[5, 10], help="severity of alert")
+            '--severity', type=str, metavar="<string>", help="severity of alert")
```
Comment (rongzhus) [inline synctl:3497]:
```suggestion
            '--severity', type=str, metavar="<string>", choices=["warning", "critical"], help="severity of alert")
```

File: synctl [hunk 2]
```diff
@@ -3651,7 +3678,7 @@ class ParseParameter:
         update_group.add_argument(
             '--alert-channel', type=str, nargs='+', metavar="id", help="alert channel id, support multiple alert channel id")
         update_group.add_argument(
-            '--severity', type=int, metavar="<int>", choices=[5, 10], help="severity of alert")
+            '--severity', type=str, metavar="<string>", help="severity of alert")
```
Comment (rongzhus) [inline synctl:3681]:
```suggestion
            '--severity', type=str, metavar="<string>", choices=["warning", "critical"], help="severity of alert")
```

**Review by rongzhus — CHANGES_REQUESTED** [review]
got an error when update `--violation-count 4`
```
synctl update alert 74PZ6WlZQn-qG0YjsEcFOQ --violation-count 4
Traceback (most recent call last):
  File "/Users/shangrz/Desktop/instana-integration/instana-public/synthetic-synctl/./synctl", line 4274, in <module>
    main()
  File "/Users/shangrz/Desktop/instana-integration/instana-public/synthetic-synctl/./synctl", line 4235, in main
    update_alert.update_violation_count(get_args.violation_count)
  File "/Users/shangrz/Desktop/instana-integration/instana-public/synthetic-synctl/./synctl", line 2694, in update_violation_count
    self.smart_alert_config["timeThreshold"]["violationsCount"] = violation_count
AttributeError: 'UpdateSmartAlert' object has no attribute 'smart_alert_config'
```

**Review by rongzhus — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -839,7 +839,7 @@ synctl create alert [options]
 --test id [id ...]                    synthetic-test id, support multiple synthetic tests id
 --name <string>                       friendly name of the Smart Alerts
 --description, -d <string>            the description of Smart Alerts
---severity <int>                      the severity of alert is either 5 (Warning), or 10 (Critical)
+--severity <string>                   the severity of alert is either Warning or Critical
```
Comment (rui1234) [inline README.md:842]: the description says Warning/Critical, but the example is warning/critical

**Pull request merged**